bin/tachyon

```diff
@@ -238,18 +239,23 @@ if [[ ${COMMAND} == "format" ]]; then
   echo "Formatting Tachyon Master @ ${TACHYON_MASTER_ADDRESS}"
   CLASS=tachyon.Format
   PARAMETER=master
-elif [[ ${COMMAND} == "formatWorker" ]]; then
+;;
+"formatWorker")
   echo "Formatting Tachyon Worker @ `hostname -f`"
   CLASS=tachyon.Format
   PARAMETER=worker
-elif [[ ${COMMAND} == "tfs" ]]; then
+;;
+"tfs")
```

Comment:
Totally agree with you, I was just analyzing TfsShell.java when I realized this, let's just forget about it. Also, it won't give us the opportunity to change MkdirShell.java to have two inputs. Sorry for it.